tests/Acceptance/Checker/DeadCodeCheckerTest.php

```diff
@@ -23,24 +21,24 @@
  * @group Checker
  * @group DeadCodeCheckerTest
  */
-class DeadCodeCheckerTest extends TestCase
+class DeadCodeCheckerTest extends ApplicationTestCase
```

Comment:
Please, add a test that covers repository and entitymanager cases. We need to be sure in the future that these cases are covered.

Reply:
it's the second part related to codecov. all tests here https://github.com/spryker-sdk/evaluator/pull/33

Reply:
I see dummy data, but there's no output check